.github/actions/deploy_debugd/action.yml

```diff
@@ -0,0 +1,92 @@
+name: Deploy debugd
+description: Deploy the Constellation debug daemon to the cluster.
```

Comment:
The debug daemon is installed in the images.
This action simply uses cdbg (and the already installed binary) to distribute the Constellation bootstrapper and additional testing binaries to the nodes.